<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QMainWindow" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Your Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	
	background-color: rgb(45, 116, 98);
border :1px solit;
border-radiux:3px;
}
QPushButton:hover{
	
	color: rgb(255, 255, 255);
	background-color: rgb(57, 143, 0);
}</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter your email and password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Phone number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>New User Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>New Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>New Phone Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_username">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	
	background-color: rgb(45, 116, 98);
border :1px solit;
border-radiux:3px;
}
QPushButton:hover{
	
	color: rgb(255, 255, 255);
	background-color: rgb(57, 143, 0);
}</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Password">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	
	background-color: rgb(45, 116, 98);
border :1px solit;
border-radiux:3px;
}
QPushButton:hover{
	
	color: rgb(255, 255, 255);
	background-color: rgb(57, 143, 0);
}</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Phone">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	
	background-color: rgb(45, 116, 98);
border :1px solit;
border-radiux:3px;
}
QPushButton:hover{
	
	color: rgb(255, 255, 255);
	background-color: rgb(57, 143, 0);
}</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>343</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_4</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_5</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_6</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_username</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_Password</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_Phone</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
